### DISCUSSION TITLE
Jans Link - adjustment does not start.

### DISCUSSION THREAD

**Discussion by mission-leben** [JanssenProject/jans#7801] [Q&A] [answered]
I can't get any further with the Flex setup in Jans link. Basically it is the same as in the "old" Gluu version. Add Active Directory Server (test is successful), add Customer Attibute and map the whole thing under Configuration. Then set the poll time and activate the whole thing. Although it accepted all this without any problem, including the activation, it does not execute the script. This is always the load runtime. Last run: [2023-07-03 08](callto:2023-07-03 08):27:28

When I do the countercheck on the CLI or TUI page, all data is entered there exactly as in the HTTP Config. 

What is noticeable is that the snapshot directory is empty. This in turn indicates that the corresponding script is not executed or aborts. How can I isolate the error here?

(V.5.0.22)

regards

Christian

**Comment by nynymike** [comment]
@shekhar16  can you take a look?

**Comment by shekhar16** [comment, marked as answer]
Hi @mission-leben
I tried the jans link and i m getting proper value of "Last run: 1709119883822" and also my /var/jans/link-snapshots folder contains all the required files. Here is the screenshot.

![image](https://github.com/JanssenProject/jans/assets/15937500/4b22997a-860e-4f9b-9f9e-7f7da7425f1f)

Can you please check the logs  is there any error or exception .
